docs/src/main/sphinx/installation/kubernetes.rst

```diff
@@ -0,0 +1,342 @@
+
+   .. code-block:: text
+
+       NAME                                               READY   STATUS    RESTARTS   AGE
+       pod/my-trino-cluster-coordinator-bfb74c98d-rnrxd   1/1     Running   0          161m
+       pod/my-trino-cluster-worker-76f6bf54d6-hvl8n       1/1     Running   0          161m
+       pod/my-trino-cluster-worker-76f6bf54d6-tcqgb       1/1     Running   0          161m
+
+       NAME                       TYPE        CLUSTER-IP    EXTERNAL-IP   PORT(S)    AGE
+       service/my-trino-cluster   ClusterIP   10.96.25.35   <none>        8080/TCP   161m
+
+       NAME                                           READY   UP-TO-DATE   AVAILABLE   AGE
+       deployment.apps/my-trino-cluster-coordinator   1/1     1            1           161m
+       deployment.apps/my-trino-cluster-worker        2/2     2            2           161m
+
+       NAME                                                     DESIRED   CURRENT   READY   AGE
+       replicaset.apps/my-trino-cluster-coordinator-bfb74c98d   1         1         1       161m
+       replicaset.apps/my-trino-cluster-worker-76f6bf54d6       2         2         2       161m
+
+   * The top three lines are pods. These are the actual Trino containers running.
+
+   * The next line is the service. This is the method Kubernetes makes internal
+     services like the one running on the coordinator accessible outside of the
+     cluster.
+
+   * The next two lines are the deployments. Deployments can have multiple pods.
+     In the case of ``my-trino-cluster-worker`` deployment, there are multiple
+     pods. Notice the ``my-trino-cluster-coordinator`` will always have one pod.
+     Deployments also contain replica sets, but also encompass logic around how
+     upgrades and maintenance are performed.
+
+   * The final two lines are the replica sets. This defines how many instances
+     of a given pod should be available during a ready state. This is the way
+     you will scale out your workers.
+
+#. If all pods, deployments, and replica sets are running and in the ready
+   state, Trino has been successfully deployed.
+
+.. note::
+
+      Unlike most Kubernetes applications, where it's better to have many small
+      pods, Trino works best with fewer pods with more resources available. Keep
+      this in mind when configuring your Kubernetes cluster.
+
+Executing queries
+-----------------
+
+The pods running the Trino containers are all running on a private network
+internal to Kubernetes. In order to access them, specifically the coordinator,
+you need to create a tunnel to the exposed service and your computer. You can do
+this by running the commands generated upon installation.
+
+#. Store the coordinator pod name in a shell variable called ``POD_NAME``.
+
+   .. code-block:: text
+
+       export POD_NAME=$(kubectl get pods --namespace default -l "app=trino,release=my-trino-cluster,component=coordinator" -o jsonpath="{.items[0].metadata.name}")
```

Comment:
I'd only keep the namespace if it was included in the `helm install` command, but it's not. Also, `-o name`, and the variable does not need to be exported.
```suggestion
       POD_NAME=$(kubectl get pods -l "app=trino,release=my-trino-cluster,component=coordinator" -o name)
```